src/actions/editPointsActions.js

```diff
@@ -28,19 +28,23 @@ const handleCopyWorklog = (points: Point[], indices: number[]) =>
     points[indices[0]].issue
   )
 
-const handleEditTimestamp = (
+const handleEditPoint = (
   points: Point[],
   indices: number[],
-  timestamp: string
+  newPoint: Point
 ) => {
   const newPoints = sortBy(
     points.map((point, i) =>
-      indices.includes(i) ? { ...point, timestamp } : point
+      indices.includes(i) ? { ...point, ...newPoint } : point
     ),
     ['timestamp']
   )
-  const myIndex = newPoints.findIndex(p => p.timestamp === timestamp)
-  return { newPoints, indices: [myIndex] }
+  return {
+    newPoints,
+    indices: newPoint.timestamp
+      ? [newPoints.findIndex(p => p.timestamp === newPoint.timestamp)]
+      : []
```

Comment:
```suggestion
      : [myIndex]
```
without it, it would enable editing a description of a synced worklog